.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
+    "python.linting.pylintEnabled": false,
+    "python.linting.mypyEnabled": true,
+    "python.linting.enabled": true
+}
```

Comment:
this should probably be gitignored, but for now could you just delete the file from this branch

Reply:
> This feels super overkill. I'm not sure why you made it so complex

"Enterprise" code